tests/observablemixin.js

```diff
@@ -739,6 +739,14 @@ describe( 'Observable', () => {
 			observable.unbind();
 		} );
 
+		it( 'should not fail when unbinding non-binded property', () => {
+			const observable = new Observable();
+
+			observable.bind( 'foo' ).to( car, 'color' );
+
+			observable.unbind( 'bar' );
```

Comment:
I'd use `expect( () => {} ).to.not.throw();` here to be consistent with other tests.

Reply:
I also reword the test name.